### ISSUE TITLE
Corus plants don't grow on your endstone

### ISSUE THREAD

**Issue by Nathan22211** [Andrew6rant/End-Stone#2]
My recommendation, retexture vanilla endstone to your endstone and make end cobblestone the modded block, as there's a possibility other issues will pop up with mods like better end with this setup

**Comment by Andrew6rant** [comment]
Oh man, it looks like Chorus generation using Vanilla's End Stone is hardcoded in a lot of places.
I'm planning on mixin-ing into the ChorusPlantFeature, ChorusFlowerBlock, and EndIslandFeature classes in the future, but I have to be careful for mod compat. It will be better than retexturing.

**Comment by ardissaps** [comment]
Yeah this would be a great idea, the could mod add cobbles_endstone new item instead and only change texture on vanilla endstone to be smooth like overworlod stone, that would be also easier on compatibility?
